### PR TITLE
subject can be null. fall back to description

### PR DESCRIPTION
**Pull request by ocke**
:koala:

subject can be null. fall back to description

/cc @zendesk/quokka

### References
 - Jira link: 

### Risks
 - None

### CODE REVIEW

**Comment by tobico** [comment]
:+1:

File: app.js [hunk 1]
```diff
@@ -64,6 +64,9 @@
 
     renderBookmarks: function(bookmarks) {
       this.bookmarks = bookmarks;
+      this.bookmarks.forEach(function(bookmark) {
+        bookmark.ticket.subject = bookmark.ticket.subject || bookmark.ticket.description;
```
Comment (danielbreves) [inline app.js]: Don't we need to truncate this? The description could be very long, couldn't it?

**Comment by danielbreves** [comment]
When will the subject be `null`? Isn't it a required field?

**Comment by itirkarp** [comment]
Single +1? maybe you should have waited for Dan's comments :wink:

**Pull request merged**